package.json

```diff
@@ -53,24 +55,27 @@
     "@types/node": "^8.10.45",
     "@types/xml2js": "^0.4.4",
     "@types/yargs": "8.0.2",
-    "@zowe/cli-test-utils": ">=7.0.0-next.202106242030 <8.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.29.0",
+    "@typescript-eslint/parser": "^4.29.0",
+    "@zowe/cli-test-utils": "^7.0.0-next.202108121907",
```

Comment:
Should the range `">=7.0.0-next.202106242030 <8.0.0"` be restored? I believe it may have been deliberately set up that way by @gejohnston

Reply:
thanks for catching it!